### PR TITLE
Adding Audio/Music support to the tracker Zamunda.net

### PR DESCRIPTION
**Pull request by eybox**
I was the one who created both Zamunda.net and Zelka.org tracker YMLs. So far they've had only Movies and TV shows, but now I'm also adding Audio/Music support for this tracker

### CODE REVIEW

File: src/Jackett.Common/Definitions/zamundanet.yml [hunk 1]
```diff
@@ -20,6 +20,11 @@
       20: Movies/DVD
       9: XXX
       49: XXX/Other
+      6: Audio
+      30: Audio/Other
+      29: Music/DVD-R
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/zamundanet.yml]: `Music/DVD-R` & `Music/Hi-Res/Vinyl` are not valid torznab categories
see https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt#L608-L695

Comment (eybox) [reply src/Jackett.Common/Definitions/zamundanet.yml]: Yeah, sorry about that. It is a copy-paste mistake I did 👍

**Comment by garfield69** [comment]
Cheers

**Comment by eybox** [comment]
Cheers, mate!

**Pull request merged**